lms/static/sass/_lms-overrides.scss

```diff
@@ -1,19 +1,44 @@
 /* header */
-.header-global {
-    border-bottom: 2px solid $gold;
-	.logo {
-		img {
+.global-header {
+    border-bottom: 2px solid $gold !important;
```

Comment:
@tomaszgy, can you check if there is a way to avoid using `!important` here and in a few places below and instead use the CSS specificity to ensure that this style is applied?